Makefile

```diff
@@ -501,6 +501,7 @@ test-v8: v8
         --no-presubmit \
         --shell-dir=$(PWD)/deps/v8/out/$(V8_ARCH).$(BUILDTYPE_LOWER) \
 	 $(TAP_V8)
+	git clean -fdxq -- deps/v8
```

Comment:
Can we assume the presence of `git`, i.e. is this target valid for running from the source tarball?

Reply:
This target depends on the `v8` target, which executes `make-v8.sh`, which already assumes the presence of git.

Reply:
But maybe replace `x` with `X`?
If I'm understanding git's logic, it'll use node's `.git` not `deps/v8/.git`

Reply:
`deps/v8/.git` is removed in `make-v8.sh` during cleanup:

https://github.com/nodejs/node/blob/088bba31a3b0222f263470fc9c95770466d16d86/tools/make-v8.sh#L16

Reply:
So definatly `-X` (it will remove explicitly ignored files, while keeping newly added files)